lib/eChronicler_web/templates/journal_entry/index.html.heex

```diff
@@ -0,0 +1,15 @@
+<h1 id="title">Journal Entries Page</h1>
+
+<div>
+ 
+    <%= for journal_entry <- @journal_entries do %>
+      <div id="entry">
+        <h3 id="title">Title: <%= journal_entry.title %></h3>
+        Author: <%= journal_entry.author %> <br>
+        Published: <%= journal_entry |> EChronicler.EChronicler.JournalEntry.format_datetime() %> <br>
+        Entry: <%= journal_entry.entry |> EChronicler.EChronicler.JournalEntry.truncate_journal_entry_text() %>
+      </div>
+      <hr>
+    <% end %>
+
+</div>
```

Comment:
Look into [semantic HTML](https://www.w3schools.com/html/html5_semantic_elements.asp)

Reply:
Done! Added `main` and `article` tags instead of the divs

Reply:
You did a great job with the semantic tags on the other template but you should also consider using them here.